src/custom/components/Button/index.tsx

```diff
@@ -22,56 +22,82 @@ import {
 
 export const ButtonPrimary = styled(ButtonPrimaryMod)`
   // CSS overrides
-  ${({ theme }) => theme.bgLinearGradient}
-  background-color: transparent;
-  border: 0;
+  ${({ theme }) => (theme.buttonPrimary?.background ? theme.buttonPrimary?.background : theme.bgLinearGradient)}
+  font-size: ${({ theme }) => (theme.buttonPrimary?.fontSize ? theme.buttonPrimary.fontSize : '16px')};
+  font-weight: ${({ theme }) => (theme.buttonPrimary?.fontWeight ? theme.buttonPrimary.fontWeight : 500)};
+  border: ${({ theme }) => (theme.buttonPrimary?.border ? theme.buttonPrimary.border : 0)};
+  box-shadow: ${({ theme }) => (theme.buttonPrimary?.boxShadow ? theme.buttonPrimary.boxShadow : 'none')};
+  border-radius: ${({ theme }) => (theme.buttonPrimary?.borderRadius ? theme.buttonPrimary.borderRadius : '9px')};
+  ${({ theme }) => theme.cursor};
+  overflow: hidden;
+  position: relative;
+
+  > div {
+    font-size: inherit;
+    font-weight: inherit;
+  }
 
   &:focus,
   &:hover,
   &:active {
-    ${({ theme }) => theme.bgLinearGradient}
-    background-color: transparent;
-    border: 0;
+    ${({ theme }) => (theme.buttonPrimary?.background ? theme.buttonPrimary?.background : theme.bgLinearGradient)}
+    border: ${({ theme }) => (theme.buttonPrimary?.border ? theme.buttonPrimary.border : 0)};
```

Comment:
no need for ternary logic its just overcrowded. set the default amount in the params (this comment applies to all of these)

Reply:
But the params are set individually in each theme. Would you suggest to create a 'general' params. object/file to set the standards for all themes? Then if the theme provides a variable, it simply overrides the general parameter.

Reply:
Alternativelly, there could be a "baseTheme" we have, and then all themes inherit for it. Then this props would be always present (so we wouldn't need ternary operators)

Reply:
Yes I would be in favor of that approach.

Reply:
Let's make this in another PR, since it feels independent from the scope of this one